### PR TITLE
Make web-page Assesible

### PR DESCRIPTION
**Pull request by quavo19**
### in this pull request;
my partner and I made some changes to make our project accessible to all, especially those with disabilities.
**here are a couple of things we took into consideration**

- page title  - no issues found
- image text alternatives - no issues found
- text headings - no issues found
- color contrast - no issues found
- resize - no issues found
- interaction - no issues found
- moving content - no issues found
- multimedia - no issues found
- the basic structure of the page - no issues found

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -426,7 +426,8 @@ textarea {
 }
 
 .button:hover {
-  background-color: #6070ff;
+  background-color: #01041b;
+  color: #f9f6f6;
```
Comment (Santiago220991) [inline style.css]: I think you did a great job here. However, there are still some things to improve. There are still errors with the contrast ratio as you can see [here](https://www.aditus.io/button-contrast-checker/raw-githack-com-2022-11-08-at-13-03-52-120?failing). Kindly fix this. :pray: :pray: 

image

image

image

**Review by Santiago220991 — CHANGES_REQUESTED** [review]
Hi @quavo19 @Franky237, 
I'm Santiago your reviewer for this PR. :wave: :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
:heavy_check_mark: Followed GitHub flow.
:heavy_check_mark: Good PR summary.
:heavy_check_mark: No linter errors.

## Required Changes ♻️
Kindly remember to add a summary about the accessibility of your project in your PR according to the guidelines. 
image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (quavo19) [reply style.css]: @Santiago220991 please the colors are from the figma design, should we still change it ?

Comment (Santiago220991) [reply style.css]: @quavo19 Yes for this project you can change the colors. Also please do not forget to add the accessibility summary in your PR. :pray: :pray: This is a project requirement. :+1: :smiley: :smiley:

File: style.css [hunk 2]
```diff
@@ -415,18 +416,19 @@ textarea {
 
 .button {
   width: 160px;
-  border: 2px solid #7f8cff;
+  border: 2px solid #2838cb;
```
Comment (TadesseDev) [inline style.css]: - [ ] Dear @team, You did great implementing the most of required changes. one thing is left regarding the contrast ratio as the previous code reviewer mentioned. Please address this to help your site be more accessible. 

### Please [use this link](https://www.aditus.io/button-contrast-checker/htmlpreview-github-io-2022-11-08-at-15-24-32-909?failing) to check for the contrast ratio.

Capture

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @team

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
199733072-bdacca19-8df9-43a7-be93-3aff2f1c15c8


 ## Highlights
✔️ Good job addressing most of the requirements. 
✔️ Followed GitHub flow.
✔️ Good PR summary.
✔️ No linter errors.

 ## Required Changes ♻️
- [ ] Hello @team, good job so far, yet as the previous code reviewer mention, your pull request should contain more detail about the accessibility of your portfolio, and most importantly a result section where you discuss your findings. 
>  - If everything was ok, you can make a statement about that (e.g. "Page titles - no issues found").
>   - If issues were found, you should link to the corresponding line of code in the Pull Request changes.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (quavo19) [reply style.css]: @Tadesse-Alemayehu can you please explain further what we have to do ?
we are really finding it very hard to understand

Comment (TadesseDev) [reply style.css]: - dear @quavo19, You are almost done. For the pull request description, I suggest you refer to [this pull request](https://github.com/Vgonma/PersonalPortfolio/pull/8). There are some details missing there also but this should be good for you to refer to. 

- And for the contrast, you just have to fix one thing. the `Get in touch` button. please [use this link](https://www.aditus.io/button-contrast-checker/htmlpreview-github-io-2022-11-08-at-15-24-32-909?failing)to review your result before submitting it for a review.

**Comment by quavo19** [comment]
### in this pull request;
my partner and I made some changes to make our project accessible to all, especially those with disabilities.
**here are a couple of things we took into consideration**

- page title  - no issues found
- image text alternatives - no issues found
- text headings - no issues found
- color contrast - no issues found
- resize - no issues found
- interaction - no issues found
- moving content - no issues found
- multimedia - no issues found
- the basic structure of the page - no issues found

**Review by TadesseDev — APPROVED** [review]
### Hi @team

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
68747470733a2f2f6669636f666f72756d732e6d796669636f2e636f6d2f74352f696d6167652f736572766572706167652f696d6167652d69642f343932363069463839303835363833444139334135422f696d6167652d73697

## Highlights
✔Good job making the requested changes.
✔️ Followed GitHub flow.
✔️ Good PR summary.
✔️ No linter errors.
## Optional suggestions
- [Optional] Hello @team. It's great you add a description to your pull request, but I notice you mention `no issues found` on all of the checks, this part was for you to discuss the kind of issue you have found and fix. and add a link to the changes you make.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**